test/unit/lib/plugins/aws/package/compile/events/websockets/lib/permissions.test.js

```diff
@@ -113,4 +113,47 @@ describe('#compilePermissions()', () => {
       },
     });
   });
+
+  it('should create a permission resource for an external authorizer function', () => {
```

Comment:
New tests should be configured via `runServerless` util, as e.g. tests in this block: https://github.com/serverless/serverless/blob/5d1c3f7ed4d481619d412b6b453d6eaf94b32b4a/test/unit/lib/plugins/aws/package/compile/events/websockets/index.test.js#L116-L258

See also: https://github.com/serverless/serverless/tree/main/test#unit-tests

Reply:
Thanks I didn't know that, was updating an older test file :)

I have now moved the new test to [test/unit/lib/plugins/aws/package/compile/events/websockets/lib/authorizers.test.js](./test/unit/lib/plugins/aws/package/compile/events/websockets/lib/authorizers.test.js) using runServerless.